backend/core/src/listings/entities/listing.entity.ts

```diff
@@ -664,6 +664,16 @@ class Listing extends BaseEntity {
   @IsDate({ groups: [ValidationsGroupsEnum.default] })
   @Type(() => Date)
   lastApplicationUpdateAt?: Date | null
+
+  /**
+   * This is only added to enable passing directly from external listings since
```

Comment:
My understanding of the slug is that it's purely cosmetic (ie just for SEO), but I wasn't sure.  I've seen routes that require slugs to consider the URL valid and some that don't, and I wasn't confident that it would always be the case on the Bloom side that it didn't really matter.  Since I also couldn't be sure that the slug generated by doorway would be identical to the slug that _might_ be needed by Bloom, I figured I'd play it safe and just keep the value that I got from the API response and propagate it through our endpoint.  That's a fair assessment though, and I'm fine taking it out in a subsequent PR